README.md

```diff
@@ -34,6 +34,16 @@ module.exports = {
       },
       requiredConfig: ['awesomeApiKey'], // throw an error if this is not configured
 
+      // return an object from here to add functionality to the `pluginHelper`
+      // that is passed in to user defined config property functions
+      pluginHelper: function(context) {
+        return {
+          sayHello: function() {
+            return 'Hello' + context.project.name();
+          }
```

Comment:
maybe something that gets added via revisions-data would be a useful example here. I assume notification plugins like slack or similiar could be interested in information that is easily accessible via helper functions like "deployerName" or "version".